deliver/lib/deliver/upload_price_tier.rb

```diff
@@ -21,7 +21,7 @@ def upload(options)
       # Need to get prices from the app's relationships
       # Prices from app's relationship doess not have price tier so need to fetch app price with price tier relationship
       app_prices = app.prices
-      if app_prices.first
+      if app_prices && app_prices.first
```

Comment:
Adding this additional check did fix this issue for me. I did not apply the change in `tunes.rb` though.